prover/protocol/coin/coin.go

```diff
@@ -117,9 +121,24 @@ func NewInfo(name Name, type_ Type, round int, size ...int) Info {
 		if len(size) > 0 {
 			utils.Panic("size for Field")
 		}
+	case FromSeed:
+		if len(size) > 0 {
+			utils.Panic("size for Field")
+		}
 	default:
 		panic("unreachable")
 	}
 
 	return infos
 }
+
```

Comment:
Why do you need this function if you already made the necessary modification in Sample?